src/nodes.coffee

```diff
@@ -3894,3 +3934,6 @@ unfoldSoak = (o, parent, name) ->
   parent[name] = ifn.body
   ifn.body = new Value parent
   ifn
+
+unwrapStringObject = (str) ->
+  str.valueOf?() ? str
```

Comment:
Shouldn’t this be implemented as `unwrap` on the `StringLiteral` class?

Reply:
@GeoffreyBooth I'm not sure this should be conflated with our concept of `unwrap()`, this is just ensuring that we normalize `String` objects to primitive strings (since I think I ran into Prettier not treating a `String` object the same) - updated the name of the helper to `normalizeStringObject()`